File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    while(true){
        height = get_int("Height: ");
        if(height > 0 && height <=8){
            break;
        }
    }
    int width = height *2 + 2;

    for(int i = 0; i < height; i++){
        for(int j = 0; j < width; j++){
            if (j < height-i-1){
                printf(" ");
            } else if (j >= height -i-1 && j < height){
                printf("#");
            } else if (j == height || j == height +1){
                printf(" ");
            } else if (j > height +1 && j <= height +i+2){
               printf("#");
            }
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
# TODO

from cs50 import get_int

while(True):
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

width = height * 2 + 2
for i in range(height):
    for j in range(width):
        if j < height-i-1:
            print(" ", end="")
        elif(j >= height - i-1 and j < height):
            print("#", end="")
        elif(j == height or j == height + 1):
            print(" ", end="")
        elif(j > height + 1 and j <= height + i+2):
            print("#", end="")
    print("")
